DiskSetup.sh

```diff
@@ -68,17 +69,15 @@ function CarveLVM() {
    fi
 
    # Gather info to diagnose seeming /boot race condition
-   grep "${BOOTLABEL}" /proc/mounts
-   # shellcheck disable=SC2181
-   if [[ $? -eq 0 ]]
+   if [[ $(grep "${BOOTLABEL}" /proc/mounts)$? -eq 0 ]]
```

Comment:
I've previously had issues with this method if I didn't suppress `grep`'s output.

Reply:
Truth. Will add -q.

Reply:
Updated.